### DISCUSSION TITLE
when run volume copy, the .dat file lastModifyTime change, is right？

### DISCUSSION THREAD

**Discussion by ck789987** [seaweedfs/seaweedfs#2302] [Q&A]
hello 
     when i run volume move，doCopyFile() create new file, copy data and rm sourcefile.  but the new file lastModifyTime changed，when reload this volume, is there any problem with ttl logic as expired() function ? 
     should i recover the sourceFile lastModifyTime (eg. run os.Chtimes ) after volume file copy finish ?

**Comment by chrislusf** [comment]
Good point. This seems a bug.

**Reply by ck789987** [reply]
thanks for answer.

**Reply by chrislusf** [reply]
Added a fix d1a4e19a3f0aeaabcf87aba3c99880f82bde87a3
